### PR TITLE
Add evm template

### PR DESCRIPTION
**Pull request by drklee3**
The evm template currently uses `agas` and not `ukava` for the evm denom as it needs 18 decimal places.

Also adds another key for "faucet" that contains a lot of `agas`, used for testing deployment of a basic smart contract.

### CODE REVIEW

File: config/templates/kava/evm/initstate/.kava/config/genesis.json [hunk 1]
```diff
@@ -0,0 +1,2454 @@
...
+                }
+              ],
+              "length": "31536000"
+            }
+          ]
+        },
+        {
+          "@type": "/ethermint.types.v1.EthAccount",
+          "base_account": {
+            "address": "kava1ak4pa9z2aty94ze2cs06wsdnkg9hsvfkp40r02",
```
Comment (DracoLi) [inline config/templates/kava/evm/initstate/.kava/config/genesis.json:216]: node: this is the eth address of the validator key, which we don't have a keyring for. Should we include one for this as well so we can send transactions via evm?

File: config/templates/kava/evm/initstate/.kava/config/client.toml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# This is a TOML config file.
+# For more information, see https://github.com/toml-lang/toml
+
+###############################################################################
+###                           Client Configuration                            ###
+###############################################################################
+
+# The network chain ID
+chain-id = ""
```
Comment (DracoLi) [inline config/templates/kava/evm/initstate/.kava/config/client.toml:9]: does this need to be populated? This sets the default chain id for the cli right?

**Review by DracoLi — APPROVED** [review]
Looks good. Everything is working properly when I tested the evm using this template.
Just the chain-id field which i believe needs to be populated.

**Comment by DracoLi** [comment]
One thing to add is that it's a bit confusing with the keys that can be used for evm and the ones that can't. Maybe worth considering adding some docs in our EVM Development Guide to include those details when using this template?

File: config/templates/kava/evm/initstate/.kava/config/genesis.json (hunk 1 above)
Comment (drklee3) [reply config/templates/kava/evm/initstate/.kava/config/genesis.json:216]: Right, I'll just include it

**Comment by drklee3** [comment]
Oh nice I just saw the guide, I will add to it 😄 I moved the page up on confluence since its hard to find stuff on there when it's buried below lol

**Pull request merged**